tests/test_isg.py

```diff
@@ -472,7 +473,42 @@ def test_set_tags(tagname, tagval, expname, expval, exptype):
 
 
 def test_write_geometry():
-    pass
+    # write a file
+    ig = isg.IsGm()
+    sname = 'write-test.h5m'
+    ig.write_geometry(sname, test_dir)
+    # check that file exists
+    exp_file = test_dir + '/' + sname
+    if isfile(exp_file):
+        r0 = True
```

Comment:
I would initialize `r0` to `False`

and maybe `r0` is not a proper name...

Reply:
`r` is the common name of the True/False results for the asserts used in all the tests so I am going to leave it as `r`.